lib/vdsm/storage/lvm.py

```diff
@@ -424,7 +433,12 @@ def _addExtraCfg(self, cmd, devices=tuple(), use_lvmpolld=True):
         else:
             allowed_devices = self._cached_devices()
 
-        dev_filter = _buildFilter(allowed_devices)
+        if USE_DEVICES:
+            if allowed_devices:
+                newcmd += ["--devices", _build_devices(allowed_devices)]
```

Comment:
It is nice that we have _build_filter() and _build_devices(), but there is really nothing to build for devices, so we can just do:

    if devices:
        newcmd += ["--devices", ",".join(devices)]